### PR TITLE
ports numerous PRs from tg related to ssTimers

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

title

## Why It's Good For The Game

i will write a better PR later

### CODE REVIEW

**Comment by triplezeta** [comment]
uhgeugheugheguhegeg

File: code/__DEFINES/subsystems.dm [hunk 1]
```diff
@@ -57,6 +57,12 @@
 ///In most cases you want a subsystem instead, so don't use this unless you have a good reason
 #define TIMER_LOOP (1<<5)
 
+///Delete the timer on parent datum Destroy() and when deltimer'd
+#define TIMER_DELETE_ME			(1<<6)
+
+///Delete the timer on parent datum Destroy() and when deltimer'd
+#define TIMER_DELETE_ME			(1<<6)
+
```
Comment (Fikou) [inline code/__DEFINES/subsystems.dm]:
```suggestion
///Delete the timer on parent datum Destroy() and when deltimer'd
#define TIMER_DELETE_ME	(1<<6)
```

File: code/__DEFINES/subsystems.dm [hunk 2]
```diff
@@ -57,6 +57,8 @@
 ///In most cases you want a subsystem instead, so don't use this unless you have a good reason
 #define TIMER_LOOP (1<<5)
 
+///Delete the timer on parent datum Destroy() and when deltimer'd
+#define TIMER_DELETE_ME	(1<<6)
```
Comment (ZephyrTFA) [inline code/__DEFINES/subsystems.dm]:
```suggestion
#define TIMER_DELETE_ME (1<<6)
```

**Comment by ZephyrTFA** [comment]
I have fixed linters like five times, if you do not pass tests this time I am getting mark to fix you

**Comment by triplezeta** [comment]
gurgles

**Comment by ZephyrTFA** [comment]
thank fucking god

**Pull request merged**